docs/open_source/release.md

```diff
@@ -82,6 +82,6 @@ This will ensure the commits *and* tags are pushed to the remote git repository.
 
 ### Deploy Catalog Server
 
-`MDC_ENV=development npm run build && gcloud app deploy`
+`MDC_ENV=development npm run build:demos && gcloud app deploy`
```

Comment:
We also need to update the "Appengine development server" command in [CONTRIBUTING.md](https://github.com/material-components/material-components-web/blob/6078784954b484df575fd54bd732445add358a96/CONTRIBUTING.md#appengine-development-server)